docker/list_operator.sh

```diff
@@ -37,7 +37,7 @@ echo "Threshold Weight: $THRESHOLD_WEIGHT"
 
 # Get current block height and calculate range
 LATEST_BLOCK=$(cast block-number --rpc-url "$LOCAL_ETHEREUM_RPC_URL")
-FROM_BLOCK=$((LATEST_BLOCK - 2000))
```

Comment:
why this change? ideally we want all registrations ever, 2000 was small, 900 is smaller.

Reply:
ETH RPC nodes limited to 1000 (999 including latest block) with multiple RPC provides i tried on holesky